src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/WPFAddItemCommandHandler.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    [ExportCommandGroup(CommandGroup.WPF)]
+    [AppliesTo(ProjectCapability.WPF)]
+    internal partial class WPFAddItemCommandHandler : AbstractAddItemCommandHandler
+    {
+        private static readonly Guid s_wpfPackage = new Guid("{b3bae735-386c-4030-8329-ef48eeda4036}");
+
+        // from VS: src\vsproject\cool\coolpkg\resource.h
+        protected enum LegacyCSharpStrings : uint
```

Comment:
Why protected?

Can we move these to a separate file? When I open the file you want to see meat of the class not the boilerplate enums/types that the thing uses.